runtime/makelib/targets.mk.win.inc.ftl

```diff
@@ -249,8 +245,8 @@ ifdef UMA_TREAT_WARNINGS_AS_ERRORS
 ifndef UMA_SUPPRESS_WARNINGS_AS_ERRORS
 CFLAGS+=-WX
 CXXFLAGS+=-WX
-  ifdef USE_MINGW
-	MINGW_CXXFLAGS+=-Werror -Wno-deprecated-declarations
+  ifdef USE_CLANG
+	CLANG_CXXFLAGS+=-Werror -Wno-deprecated-declarations -Wno-comment -Wno-ignored-attributes
```

Comment:
Are the `-Wno-comment` and `-Wno-ignored-attributes` options required for the current use of clang to compile the bytecode interpreters? If they are needed, can we change the code to remove the warnings instead?

Reply:
These two errors were captured by Clang.

1) -Wno-comment
It was caused by  "/*" in the comment at  /runtime/oti/clang/stddef.h
```
 *		All public headers: include/<star>, arch/<star>/bits/<star>
 *		Startup files: crt/* <-------
 */
```
Add -Wno-comment to let Clang ignore  the **/*** style in checking comment.

2)-Wno-ignored-attributes
The error came from as follows:
```
In file included from ../oti\j9.h:45:
../include\jni.h:227:11: error: stdcall calling convention ignored on variadic function [-Werror,-Wignored-attributes]
        jobject (JNICALL * NewObject)(JNIEnv *env, jclass clazz, jmethodID methodID, ...);
                 ^
../include/jniport.h:29:17: note: expanded from macro 'JNICALL'
#define JNICALL __stdcall
                ^
In file included from BytecodeInterpreter.cpp:24:
```
I don't think we should change the code/logic to fit the stricter compilation rules forced by Clang at this point.

Reply:
Please change stddef.h from `/*` to `/<start>` and remove the -Wno-comment

@DanHeidinga are you ok with the -Wno-ignored-attributes ? This doesn't seem ideal. We could probably separate out what actually needs to be included, but may take some time.

Reply:
> Please change stddef.h from /* to /<star> and remove the -Wno-comment

will update and compile again to make sure it works.

Reply:
I think we need to pay attention to the warning about `__stdcall` (which is only relevant for 32-bit ABI): it's a sign the code is not reasonable. The message suggests the compiler is just doing the right thing, so I think the fix could wait (a little while).

Reply:
yes, agreed, I believe Dan does as well.

@ChengJin01 the -Wno-ignored-attributes option needs to be removed. This probably means including more specifically what is required to build the files compiled with clang, so that jni.h doesn't get included.

Reply:
You an also try compiling an example program that uses NewObject with msvc2010, with and without the __stdcall declaration. Look at the code which is generated, and see if its any different.

Reply:
Will figure out what needs to be modified to get this compiled  without-Wno-ignored-attributes